main.go

```diff
@@ -254,30 +309,60 @@ func _main() error {
 	if err != nil {
 		return err
 	}
+	projectID := projectTmp.Name()
+	if cache {
+		stdinTmpCopy := false
+		projectID, stdinTmpCopy, err = getFilesContentHash(files, projectTmp.Name())
+		if err != nil {
+			fmt.Printf("Error creating hash for cache mode: %v, defaulting to normal mode", err)
```

Comment:
Everywhere you're using fmt.Println you should use log.Println instead. This is because fmt.Println goes to stdout and log.Println goes to stderr. So if someone does shell redirection `dsq blah blah > x.json` fmt.Println stuff goes to x.json whereas log.Println stuff doesn't unless you do `dsq blah blah 2&>1 > x.json`.